lib/Expr/ExprUtil.cpp

```diff
@@ -104,6 +104,22 @@ class SymbolicObjectFinder : public ExprVisitor {
     : objects(_objects) {}
 };
 
+
+ExprVisitor::Action ConstantArrayFinder::visitRead(const ReadExpr &re) {
+  const UpdateList &ul = re.updates;
+
+  // XXX should we memo better than what ExprVisitor is doing for us?
```

Comment:
Not a fan `XXX`. Please don't use it

Reply:
https://www.snellman.net/blog/archive/2017-04-17-xxx-fixme/

Reply:
I just use `XXX`, because its copied from other code in that file. So I guess I should change all of them?

Reply:
I think it's better to just update them as we go along. Otherwise `git blame` becomes very hard to use.